slides/12.qmd

```diff
@@ -158,29 +173,47 @@ encapsulated_OOP --> RC
 - **R6**
   - Paradigm: encapsulated OOP
   - Noteworthy: resolves issues with RC
-- **R7**
+- **S7**
```

Comment:
This should go in the functional section (looks like a previous cohort put it in the wrong place).